docs/tdvt.md

```diff
@@ -176,132 +174,134 @@ This is a selection of the output:
+The Connection Tests, and any other tests with the attribute `SmokeTest = True`, are run before the other tests.
+They can be run by themselves using the --verify flag (e.g. tdvt run postgres --verify).
+
+Option | Example Value | Default | Description | Required?
+-|-|-|-|-
+CastCalcsTestEnabled | True | True | Runs connection smoke test against calcs table if true | No
+StaplesTestEnabled | True | True | Runs connection smoke test against Staples table if true | No
+
+## Example INI File
+This is a basic ini file that runs tests against our postgres_jdbc sample connector. Here, we use an already existing logical query format, skip some tests, and run connection smoke tests before the rest of the test groups.
 
 ```ini
 [Datasource]
-
-Name = your_datasource_name  #i.e. mydb
-
-CommandLineOverride = -DLogLevel=Debug #Space separated list of arguments that are passed through unchanged to tabquerytool. Most Tableau arguments require a prepended '-D'.
-#If you are testing a connector, be sure to add the command line argument -DConnectPluginsPath and have it pointed at the folder where your connector is located
-
-MaxThread = 6   #You can add this to control Max Thread number when you use TDVT to run single datasource, it cannot apply with multi datasource
-
-TimeoutSeconds = 6000 # You can overload the timeout for queries if your queries are running slow. By default this is set to 3600.
-
-LogicalQueryFormat = dbo
-
-#You can add a new logical config here and use it above. These are example attributes; you wouldn't set them all since some are mutually exclusive.
-
-[LogicalConfig]
-Name = my_logical_query
-tablename = SomePreix_$dsName #$dsName will be substituted with Calcs or Staples.
-tablePrefix = [MySchema].
-tablePostfix = [MySchema].
-tablenameUpper = True
-tablenameLower = True
-bool_underscore = True
-fieldnameDate_underscore = True
-fieldnameLower = True
-fieldnameUpper = True
-fieldnameNoSpace = True
-fieldnameLower_underscore = True
-fieldnameUnderscoreNotSpace = True
-
+Name = postgres_jdbc
+LogicalQueryFormat = simple_public
+CommandLineOverride = -DLogLevel=Debug -DConnectPluginsPath=plugins
 
 [StandardTests]
-
-#You can put in comma-separated string values that match part or all of a test name to exclude them. The asterisk works as a wildcard.
-
-LogicalExclusions_Calcs = string.right
-
-LogicalExclusions_Staples = Filter.Trademark
-
-ExpressionExclusions_Standard = string.ascii,string.char,logical
-
-#If you remove this section then your test sute will not include the Level of Detail tests.
+ExpressionExclusions_Standard = string.isdate,date.datediff.day,date.datediff.dayofyear,date.datediff.hour,date.datediff.minute,date.datediff.second,date.datediff.sow.day,date.datediff.sow.dayofyear,date.datediff.weekday,date.datediff.sow.hour,date.datediff.sow.minute,date.datediff.sow.second,date.datediff.sow.weekday,datetime.cast.float
```

Comment:
Maybe shorten this example slightly or use the `*` notation

Reply:
I'll take a few of them out, thanks